### ISSUE TITLE
Instant Search: Improve keyboard navigation of expanded results.

### ISSUE THREAD

**Issue by a8ck3n** [Automattic/jetpack#25698]
### Impacted plugin

Search

### What

Each result has three redundant anchor elements. They should behave normally in a visual context but should be ignored when using keyboard navigation or screen readers.

### How

The second and third links (the path/breadcrumb and the image) should be removed from the tab loop and hidden from screen readers.

![RedundantAnchorElements](https://user-images.githubusercontent.com/40267301/185370777-6149a828-930e-4dd6-beb3-8ca1edb87362.png)

### To Test

1. Visit the [Search Overlay on Jetpack.com](https://jetpack.com/?s=features&category=tips-tricks&result_format=expanded).
2. Tab from the text input field through to the first result.
3. Tab through to the second result. Notice how we have three links per result.

**Note:** The sort options (relevance, newest, oldest) might not highlight. Will be addressed separately.
